code/__HELPERS/game.dm

```diff
@@ -380,3 +380,53 @@
 
 	return new /datum/projectile_data(src_x, src_y, time, distance, power_x, power_y, dest_x, dest_y)
 
+/proc/pollCandidates(var/Question, var/jobbanType, var/datum/game_mode/gametypeCheck)

```

Comment:
make sure to go through and make all the other event systems that use this shit but copypasted use this proc instead